x-pack/plugins/observability_solution/inventory/public/components/entities_grid/entity_name/index.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiLink, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React, { useCallback } from 'react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
```

Comment:
I've noticed this now, but maybe this `useKibana` import can be replaced with `inventory`'s version which will have the correct deps/services. At least if this particular component will remain in `inventory`